lib/protocol/sequences/ChangeUser.js

```diff
@@ -7,29 +7,28 @@ module.exports = ChangeUser;
 Util.inherits(ChangeUser, Sequence);
 function ChangeUser(options, callback) {
   Sequence.call(this, options, callback);
-
-  this._user          = options.user;
```

Comment:
The packets are publicly exposed. Even though these are `_`-prefixed properties, historically when we change these people break; please keep all the old properties, based on our past experiences.